tests/check_simple_example_c.sh

```diff
@@ -60,8 +60,16 @@ else
 
     rm -f testbin
 
+    # limit cpu time and stack size for executing test
+    cpu_time_limit=$(ulimit -t)
+    stack_limit=$(ulimit -s)
+    ulimit -S -t 120 -s 1024
```

Comment:
This also limits the stack for the jvm process but if the jvm still works with just 1MB of stack I see no real harm doing that.